docs/sentry-setup.md

```
@@ -0,0 +1,117 @@
+- Display names (redacted by `_sanitizeEvent`)
+- IP addresses (can be disabled in Sentry project settings)
+- Bluetooth MAC addresses
+- Any content from the frequency room (audio, messages, etc.)
+
+## Testing
+
+To test crash reporting locally:
+
+1. Set the `SENTRY_DSN` environment variable
+2. Enable crash reporting via SQLite:
+   ```dart
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)